schemas/summary/dd_summary.xsd

```diff
@@ -4034,12 +4058,21 @@
 		<xs:sequence>
 			<xs:element name="time" type="summary_constant_flt_0d">
 				<xs:annotation>
-					<xs:documentation>Time of the disruption</xs:documentation>
+					<xs:documentation>Time of the disruption =  min(t_TQ, t_VDE), where t_TQ and t_VDE are the times at which the Thermal Quench (TQ) and the Vertical Displacement Event (VDE) are detected by their respective detectors</xs:documentation>
 					<xs:appinfo>
 						<units>s</units>
 					</xs:appinfo>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="time_predisruption" type="summary_constant_flt_0d">
+				<xs:annotation>
+					<xs:documentation>Pre-disruption time = disruption.time - t_pre, where t_pre[s] = 0.0025 * R0[m]^2 and R0 is the major radius where the toroidal field is defined (../../global_quantities/r0)</xs:documentation>
```

Comment:
Is this a strict definition (if so, no need to store since it can be calculated from the `disruption.time` and `R0`) or is it more  an expression of how it is typically/approximately given?  I ask since `R0` itself is not strictly defined and it's left to each device to define the reference position on which they give `B0`.
My suggestion would be to remove the equality sign in this statement and replace it with something like "approximately given by".